app/grandchallenge/core/templates/grandchallenge/partials/filters.html

```diff
@@ -8,7 +8,7 @@
        data-target="#filterForm"
        aria-expanded="false"
        aria-controls="filterForm">
-        <i class="fas fa-filter"></i>&nbsp;Filter {{ filter.Meta.model|meta_attr:'verbose_name_plural'|title }}
+        <i class="fas fa-filter"></i>&nbsp;Filter  {% if not model_name %}{{ filter.Meta.model|meta_attr:'verbose_name_plural'|title }}{% else %} {{ model_name|title }}s {% endif %}
```

Comment:
I made these changes here, because otherwise it would say `Filter Follows`, which I thought might be confusing for the user. But maybe it's a bit of a convoluted code change for such little benefit...

Reply:
Ah, yeah, that makes sense. But since the Follow model is from actstream, this would mean overriding the actstream model, but if I get it to work that should be fine.